devices/flash-zynq7000/flashdrv.c

```diff
@@ -195,10 +204,10 @@ static int flashdrv_sectorErase(addr_t offs)
 
 	switch (CFI_SIZE_SECTION(cfi->regs[id].size)) {
 		case 0x1000:
-			cmd = &fdrv_common.info.cmds[flash_cmd_p4e];
+			cmd = (fdrv_common.info.addrMode == flash_4byteAddr) ? &fdrv_common.info.cmds[flash_cmd_4p4e] : &fdrv_common.info.cmds[flash_cmd_p4e];
```

Comment:
Slightly shorter version, I don't know if prettier
```suggestion
			cmd = &fdrv_common.info.cmds[(fdrv_common.info.addrMode == flash_4byteAddr) ? flash_cmd_4p4e : flash_cmd_p4e];
```

Reply:
It's just slightly shorter, so I leave it as it is.